src/tools/dashboard/components/styles/Dashboard.scss

```diff
@@ -237,7 +245,8 @@
         grid-template-areas:
           'type name   time'
           '.    status time'
-          'id   id     actions';
+          'id   id     actions'
+          'parameters parameters .';
         grid-template-columns: 10ch minmax(27ch, auto) 12ch;
         grid-gap: 0;
         grid-template-rows: 1fr 1fr 1fr;
```

Comment:
Haven't had a chance to test yet, but here since you've added a row it might be good to be explicit about the sizing of that row by adding a fourth value. I'm assuming this should be `auto` and not `1fr`, the point being that it shouldn't have any height if no parameter is displayed

Reply:
You are right. I set it to auto and tested. It takes space only when there is content